ProcessLib/NeumannBc.h

```diff
@@ -53,15 +53,12 @@ class NeumannBc
         NeumannBcConfig const& bc,
         unsigned const integration_order,
         AssemblerLib::LocalToGlobalIndexMap const& local_to_global_index_map,
-        std::size_t const component_index,
-        MeshLib::MeshSubset const& mesh_subset_all_nodes
-        )
-        :
-          _function(*bc.getFunction()),
-          _all_mesh_subsets(local_to_global_index_map.getNumComponents(), nullptr),
+        std::size_t const component_id,
```

Comment:
Probably now you should document that this is the component of the variable. And maybe it would be more natural to exchange these two arguments.